spark/src/main/kotlin/com/adevinta/spark/components/snackbars/SnackbarHost.kt

```diff
@@ -220,147 +174,3 @@ public class SnackbarHostState {
         }
     }
 }
-
-internal fun SnackbarDuration.toMillis(
```

Comment:
I don't understand why we moved the snackbar animation handled in the `SnackbarHost` outside of the same file, can we discuss this?